FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder/QueryBuilderFactory.php

```diff
@@ -148,6 +149,20 @@ private function addSorting(Criteria $criteria, QueryBuilder $query, ShopContext
         }
     }
 
+    /**
+     * @param QueryBuilder $query
+     */
+    private function addPushAttribs(QueryBuilder $query)
```

Comment:
As we still support PHP 5.6, please remove the typehint. In the docblock it should still stay though.

Reply:
Aren't typehints of classes supported in PHP 5.6?
https://mlocati.github.io/articles/php-type-hinting.html

Reply:
You are of course totally correct. My bad, haven't used such a legacy version in a while. Was just thinking about it and that's why I commented.